docs/limitations.rst

```diff
@@ -11,9 +14,43 @@ certain limitations:
 
 - The NumPy interface ``pybind11::array`` greatly simplifies accessing
   numerical data from C++ (and vice versa), but it's not a full-blown array
-  class like ``Eigen::Array`` or ``boost.multi_array``.
+  class like ``Eigen::Array`` or ``boost.multi_array``. ``Eigen`` objects are
+  directly supported, however, with ``pybind11/eigen.h``.
+
+Large but useful features could be implemented in pybind11 but would lead to a
+significant increase in complexity. Pybind11 strives to be simple and compact.
```

Comment:
Nitpicking: does "pybind11" take a capital at the start of a sentence?

Reply:
Yes, at least it does in 

```
docs/advanced/cast/eigen.rst:binding functions that change their (Matrix) arguments.  Pybind11 allows you to
docs/advanced/cast/functional.rst:    case, there is no overhead. Pybind11 will extract the underlying C++
docs/advanced/classes.rst:type via a holder type argument to ``class_``. Pybind11 provides a helper class
docs/advanced/functions.rst:Pybind11 provides a ``py::kw_only`` object that allows you to implement
docs/cmake/index.rst:Pybind11 can be used with ``add_subdirectory(extern/pybind11)``, or from an
docs/compiling.rst:    from pybind11.setup_helpers import Pybind11Extension
docs/compiling.rst:Pybind11 supports modern CMake usage patterns with a set of interface targets,
docs/faq.rst:   3.18.2+ best). Pybind11 in these cases uses the new CMake FindPython instead
docs/faq.rst:3. Set ``PYBIND11_NOPYTHON`` to ``TRUE``. Pybind11 will not search for Python.
docs/limitations.rst:significant increase in complexity. Pybind11 strives to be simple and compact.
tools/pybind11NewTools.cmake:      "${_Python} was found without the Interpreter component. Pybind11 requires this component.")
```

I didn't write all of those (though I did write some of them).

Reply:
OK, fine, then